db/seeds.rb

```diff
@@ -11,13 +11,13 @@ def load_data
     create_cropbot
     load_crops
     load_plant_parts
+    load_paid_account_types
+    load_products
 
     # for development and staging environments only
```

Comment:
you need to change the comment back as well ;)

and maybe leave a note saying NOT to create these accounts on staging because they have publicly known passwords.